addons/storyshots/storyshots-core/src/api/StoryshotsOptions.ts

```diff
@@ -0,0 +1,23 @@
+import { IOptions } from 'glob';
+import { Stories2SnapsConverter } from '../Stories2SnapsConverter';
+import { SupportedFramework } from '../frameworks';
+import { RenderTree } from '../frameworks/Loader';
+
+export interface StoryshotsOptions {
+  asyncJest?: boolean;
+  config?: (options: any) => void;
+  integrityOptions?: IOptions | false;
+  configPath?: string;
+  suite?: string;
+  storyKindRegex?: RegExp | string;
+  storyNameRegex?: RegExp | string;
+  framework?: SupportedFramework;
+  test?: (story: any, context: any, renderTree: RenderTree, options?: any) => any;
```

Comment:
@ndelangen @kroeder are there already existing types somewhere for `story` and `context`?

Reply:
Yes, in lib/addons